src/targets/rtr.rs

```diff
@@ -274,23 +277,27 @@ impl Source {
         }
     }
 
-    fn update(&self, update: &payload::Update) {
-        let data = self.data.load();
+    fn update(&self, update: UnitUpdate) -> bool {
```

Comment:
I would suggest documenting what the returned boolean signifies.

Reply:
Done.